application/src/main/java/org/togetherjava/tjbot/commands/help/UserBannedDeleteRecentThreadsListener.java

```diff
@@ -40,11 +42,12 @@ public UserBannedDeleteRecentThreadsListener(Database database) {
 
     @Override
     public void onGuildBan(GuildBanEvent event) {
-        RestAction.allOf(getRecentHelpThreads(event.getUser()).stream()
-            .map(event.getJDA()::getThreadChannelById)
-            .filter(Objects::nonNull)
-            .map(ThreadChannel::delete)
-            .toList()).queue();
+        getRecentHelpThreads(event.getUser()).forEach(channelId -> event.getJDA()
+            .getThreadChannelById(channelId)
+            .delete()
```

Comment:
ur `map(not null)` was quite good, id keep that. otherwise u get a NPE right now still.

Reply:
i tried to keep it but i dont know how to

```java
        getRecentHelpThreads(event.getUser()).stream()
            .map(event.getJDA()::getThreadChannelById)
            .filter(Objects::nonNull)
            .map(ThreadChannel::delete)
            .forEach(restAction -> restAction.queue(any -> {}, failure -> {}));
```

here like this i wont be able to get the thread id when it fails so no way to log it

Reply:
u dont have to log threads that are null. this is an expected situation that will happen frequently.

we only have to log when `delete` failed.

Reply:
done